packages/ow-ui/src/components/PasswordInput/Password.tsx

```diff
@@ -0,0 +1,59 @@
+import { cva } from "cva"
+import { forwardRef, HTMLInputTypeAttribute, ReactComponentElement, ReactElement, useState } from "react"
+import { Label } from "@radix-ui/react-label"
+import { IconEyeCheck, IconEyeOff} from "@tabler/icons"
+
+
+
+export interface InputProps extends React.HTMLProps<HTMLInputElement> {
+    placeholder?: string
+    label?: string
+    withAsterisk?: boolean
+    error?: boolean | string
+    inputInfo?: string
+    eyeColor: "default" | "slate" | "gray"
+  }
+
+export const PasswordInput = forwardRef<HTMLInputElement, InputProps>(({label, withAsterisk, error, inputInfo, eyeColor, ...props}, ref) => {
+  const [visible,setVisibility] = useState(false)
+  const Icon = visible ? <IconEyeOff onClick={() => setVisibility(visibility => !visibility)}/> : <IconEyeCheck onClick={() => setVisibility(visibility => !visibility)}/>;
+  const InputType = visible ? "text" : "password";
+  
+  return (
+        <div className="flex flex-col">
+            {label && (
+            <Label htmlFor={props.id} className="mb-2">
+              {label} {withAsterisk && <span className="text-red-11">*</span>}
+            </Label>
+            )}
+            <span className="text-xs">{inputInfo}</span>
+            <div className="relative">
+              <input type={InputType} {...props} ref={ref} className={input({error: !!error})} />
+                <div>
+                    <span className={eye({color: eyeColor})}>{Icon}</span>
```

Comment:
Then we pass the props here instead
```suggestion
                    <span className={eye({color: eyeColor})}>
                      <Icon onClick={() => setVisibility(visibility => !visibility)} />
                    </span>
```